scripts/utils/utils.tcl

```diff
@@ -496,5 +406,257 @@ proc cat {args} {
     return $res
 }
 
+proc manipulate_layout {args} {
+    # Requires at least one non-flag/option arg, which is the path to the script.
+    set options {
+        {-indexed_log optional}
+        {-input optional}
+        {-output optional}
+        {-output_def optional}
+    }
+
+    set flags {}
+
+    parse_key_args "manipulate_layout" args arg_values $options flag_map $flags
+
+    set_if_unset arg_values(-indexed_log) /dev/null
+    set_if_unset arg_values(-input) $::env(CURRENT_ODB)
+    set_if_unset arg_values(-output) $arg_values(-input)
```

Comment:
so the default output is inplace change of the odb file?

Reply:
yep